content/community/organization/policies.md

```diff
@@ -0,0 +1,110 @@
++++
+type = "article"
+title = "Policies"
+date = "2020-11-21T12:00:00.000Z"
+tags = []
++++
+
+The standards that govern the Haiku community are similar to the *Unwritten Constitutions* that govern some countries, where their constitution simply acts as an aggregation of other laws, rather than a standalone constitution.
+
+## Glossary
+
+|Term|Meaning|
+|---|---|
+|Community Member|The term "community member" is used in this document to refer to any individual who is engaged with the Haiku project in any way.|
+|Communications Channel|The term "communications channel" is intended to cover all communications channels related to the Haiku project.  Examples of "communications channels" include IRC, mailing lists, emails, code-reviews, discussion boards and other media.|
+|Development Team|The term "development team" is used to refer to a group of individuals who have commit access into the Haiku source code repository and who are recently actively engaged on the project contributing to the source code.|
+
+## Volunteers
```

Comment:
I would rather have this and the definition of "development team" moved into the "governance" page.